```
Successfully installed lxml-stubs-0.5.1 types-Markdown-3.7.0.20241204 types-PyMySQL-1.1.0.20241103 types-Pygments-2.19.0.20250305 types-aiofiles-24.1.0.20241221 types-colorama-0.4.15.20240311 types-docutils-0.21.0.20241128 types-oauthlib-3.2.0.20240806 types-pywin32-309.0.0.20250311 types-simplejson-3.20.0.20250218 types-tzlocal-5.1.0.1 types-ujson-5.10.0.20240515
core/helper/ssrf_proxy.py:23: error: Name "s" is not defined  [name-defined]
core/helper/ssrf_proxy.py:55: error: Argument "verify" to "Client" has incompatible type "bool | None"; expected "str | bool | SSLContext"  [arg-type]
Installing missing stub packages:
core/helper/ssrf_proxy.py:62: error: Argument "verify" to "Client" has incompatible type "bool | None"; expected "str | bool | SSLContext"  [arg-type]
core/helper/ssrf_proxy.py:65: error: Argument "verify" to "Client" has incompatible type "bool | None"; expected "str | bool | SSLContext"  [arg-type]
/home/runner/work/dify/dify/api/.venv/bin/python -m pip install lxml-stubs types-Markdown types-PyMySQL types-Pygments types-aiofiles types-colorama types-oauthlib types-protobuf types-pywin32 types-simplejson types-tzlocal types-ujson
```

--- a/api/tests/unit_tests/core/workflow/nodes/http_request/test_http_request_node.py
+++ b/api/tests/unit_tests/core/workflow/nodes/http_request/test_http_request_node.py
@@ -197,7 +197,6 @@ def test_http_request_node_form_with_file(monkeypatch):
 
 
 def test_http_request_node_form_with_multiple_files(monkeypatch):
-    """测试表单中上传多个文件的情况"""
     data = HttpRequestNodeData(
         title="测试多文件上传",
         method="post",
@@ -222,13 +221,11 @@ def test_http_request_node_form_with_multiple_files(monkeypatch):
         ),
     )
 
-    # 创建变量池并添加文件数组变量
     variable_pool = VariablePool(
         system_variables={},
         user_inputs={},
     )
 
-    # 创建多个文件对象
     files = [
         File(
             tenant_id="1",
@@ -250,7 +247,6 @@ def test_http_request_node_form_with_multiple_files(monkeypatch):
         ),
     ]
 
-    # 添加文件数组变量到变量池
     variable_pool.add(
         ["1111", "files"],
         ArrayFileVariable(
@@ -259,7 +255,6 @@ def test_http_request_node_form_with_multiple_files(monkeypatch):
         ),
     )
 
-    # 创建节点
     node = HttpRequestNode(
         id="1",
         config={
@@ -294,22 +289,18 @@ def test_http_request_node_form_with_multiple_files(monkeypatch):
         ),
     )
 
-    # 模拟文件下载函数
     monkeypatch.setattr(
         "core.workflow.nodes.http_request.executor.file_manager.download",
         lambda file: b"test_image_data" if file.mime_type == "image/jpeg" else b"test_pdf_data",
     )
 
-    # 验证请求参数
     def attr_checker(*args, **kwargs):
-        assert kwargs["data"] == {"description": "多文件测试"}
+        assert kwargs["data"] == {"description": "multiple files test"}
 
-        # 验证文件列表
         assert len(kwargs["files"]) == 2
-        assert kwargs["files"][0][0] == "files"  # 第一个文件的键名
-        assert kwargs["files"][1][0] == "files"  # 第二个文件的键名
+        assert kwargs["files"][0][0] == "files"
+        assert kwargs["files"][1][0] == "files"
 
-        # 验证文件内容和类型
         file_tuples = [f[1] for f in kwargs["files"]]
         file_contents = [f[1] for f in file_tuples]
         file_types = [f[2] for f in file_tuples]
@@ -326,7 +317,6 @@ def test_http_request_node_form_with_multiple_files(monkeypatch):
         attr_checker,
     )
 
-    # 执行节点并验证结果
     result = node._run()
     assert result.status == WorkflowNodeExecutionStatus.SUCCEEDED
     assert result.outputs is not None

--- a/api/tests/unit_tests/core/workflow/nodes/http_request/test_http_request_node.py
+++ b/api/tests/unit_tests/core/workflow/nodes/http_request/test_http_request_node.py
@@ -198,7 +198,7 @@ def test_http_request_node_form_with_file(monkeypatch):
 
 def test_http_request_node_form_with_multiple_files(monkeypatch):
     data = HttpRequestNodeData(
-        title="测试多文件上传",
+        title="test",
         method="post",
         url="http://example.org/upload",
         authorization=HttpRequestNodeAuthorization(type="no-auth"),
@@ -213,9 +213,9 @@ def test_http_request_node_form_with_multiple_files(monkeypatch):
                     file=["1111", "files"],
                 ),
                 BodyData(
-                    key="description",
+                    key="name",
                     type="text",
-                    value="多文件测试",
+                    value="test",
                 ),
             ],
         ),
@@ -295,7 +295,7 @@ def test_http_request_node_form_with_multiple_files(monkeypatch):
     )
 
     def attr_checker(*args, **kwargs):
-        assert kwargs["data"] == {"description": "multiple files test"}
+        assert kwargs["data"] == {"name": "test"}
 
         assert len(kwargs["files"]) == 2
         assert kwargs["files"][0][0] == "files"

